crates/rsharp/cs/src/RSharp.cs

```diff
@@ -9,7 +9,7 @@ public class Metadata
 #if UNITY_IOS && !UNITY_EDITOR
         internal const string LIBRARY_NAME = "__Internal";
 #else
-        internal const string LIBRARY_NAME = "unity_states";
+        internal const string LIBRARY_NAME = "constellation";
```

Comment:
For now, leave the name as-is, and I'll try to prioritize a better solution that avoids needing a single dll to target

Reply:
Reverting to `unity_states`. The Movieoke contracts PRs will be blocked until we have a solution for this.